### ISSUE TITLE
 Content Guidelines - introduction

### ISSUE THREAD

**Issue by SamHyler** [hackforla/expunge-assist#1347]
### Overview
Updating the Content Guidelines piece by piece
### Action Items
- [ ] Review the introduction
- [ ] Discuss what needs updating
- [ ] Make iteration
- [ ] Present in Content for feedback and discussion
- [ ] Iterate

### Resources/Notes
Work in Figma

**Comment by emmathrash** [comment]
@SamHyler I don't think this issue needs to much, so I want to say it's done? I would argue that it may be one of the last on the timeline CG issues, which we can consider when editing the epic.
